gitlab_test.go

```diff
@@ -16,6 +16,228 @@ import (
+            "committer_name": "Administrator",
+            "committer_email": "admin@example.com",
+            "committed_date": "2019-01-03T01:53:28.000Z"
+          },
+          "assets": {
+            "count": 2,
+            "sources": [
+              {
+                "format": "zip",
+                "url": "http://localhost:3000/archive/v0.1/awesome-app-v0.1.zip"
+              },
+              {
+                "format": "tar.gz",
+                "url": "http://localhost:3000/archive/v0.1/awesome-app-v0.1.tar.gz"
+              }
+            ],
+            "links": []
+          }
+        }
+]`
+
+	exampleReleaseRsp = `{
+        "tag_name": "v0.1",
+        "description": "description",
+        "name": "Awesome app v0.1 alpha",
+        "description_html": "description_html",
+        "created_at": "2019-01-03T01:55:18.203Z",
+        "author": {
+          "id": 1,
+          "name": "Administrator",
+          "username": "root",
+          "state": "active",
+          "avatar_url": "https://www.gravatar.com/avatar/",
+          "web_url": "http://localhost:3000/root"
+        },
+        "commit": {
+          "id": "f8d3d94cbd347e924aa7b715845e439d00e80ca4",
+          "short_id": "f8d3d94c",
+          "title": "Initial commit",
+          "created_at": "2019-01-03T01:53:28.000Z",
+          "parent_ids": [],
+          "message": "Initial commit",
+          "author_name": "Administrator",
+          "author_email": "admin@example.com",
+          "authored_date": "2019-01-03T01:53:28.000Z",
+          "committer_name": "Administrator",
+          "committer_email": "admin@example.com",
+          "committed_date": "2019-01-03T01:53:28.000Z"
+        },
+        "assets": {
+          "count": 2,
+          "sources": [
+            {
+              "format": "zip",
+              "url": "http://localhost:3000/archive/v0.1/awesome-app-v0.1.zip"
+            },
+            {
+              "format": "tar.gz",
+              "url": "http://localhost:3000/archive/v0.1/awesome-app-v0.1.tar.gz"
+            }
+          ],
+          "links": []
+        }
+}`
+	exampleTagName = "v0.1"
+
+	// Runners
+	exampleDetailRsp = `{
+	"active": true,
+	"architecture": null,
+	"description": "test-1-20150125-test",
+	"run_untagged": true,
+	"id": 6,
+	"is_shared": false,
+	"contacted_at": "2016-01-25T16:39:48.066Z",
+	"name": null,
+	"online": true,
+	"status": "online",
+	"platform": null,
+	"projects": [
+		{
+			"id": 1,
+			"name": "GitLab Community Edition",
+			"name_with_namespace": "GitLab.org / GitLab Community Edition",
+			"path": "gitlab-ce",
+			"path_with_namespace": "gitlab-org/gitlab-ce"
+		}
+	],
+	"token": "205086a8e3b9a2b818ffac9b89d102",
+	"revision": null,
+	"tag_list": [
+		"ruby",
+		"mysql"
+	],
+	"version": null,
+	"access_level": "ref_protected",
+	"maximum_timeout": 3600,
+	"locked": false
+}`
+
+	exampleRegisterNewRunner = `{
+	"id": 12345,
+	"token": "6337ff461c94fd3fa32ba3b1ff4125"
+}`
+)
```

Comment:
Ow, this is quite a list. Didn't expect that to be honest 😊 In this case then maybe we should move this into it's own file? `fixtures_test.go` or something like that?

Reply:
Yes, it is. To be honest, I would keep them where they are and drop my last commit. It is better than moving all of them into one file. Thoughts?

Reply:
I think its better to group them in one location as otherwise you have to do a search each time you want to look at one of them. And finding them in somewhat random locations could be confusing...

Reply:
Lets see now, please